### PR TITLE
doc: zigbee QSG demo

### PR DESCRIPTION
**Pull request by greg-fer**
Added Zigbee quick start guide.

### CODE REVIEW

File: doc/nrf/ug_zigbee_qsg.rst [hunk 1]
```diff
@@ -0,0 +1,216 @@
+.. _ug_zigee_qsg:
+
+Zigbee quick start guide
+########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This guide is a quick demonstration of some of the basic concepts of the Zigbee network using the |NCS| and the |VSC| extension.
```
Comment (sebastiandraus) [inline doc/nrf/ug_zigbee_qsg.rst:10]: What about adding link to nRF Connect for Visual Studio Code extension?

File: doc/nrf/ug_zigbee_qsg.rst [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+   After some time, it turns off and the Zigbee network needs to be reopened.
+#. Add the light switch application.
+   Repeat steps 2 to 4 using a custom application location and name, for example ``Zigbee_light_switch``.
+   The path for the application template is :file:`nrf/samples/zigbee/light_switch`.
+#. Add the light bulb application.
+   Repeat steps 2 to 4 using a custom application location and name, for example ``Zigbee_light_bulb``.
+   The path for the application template is :file:`nrf/samples/zigbee/light_bulb`.
+#. Press **Button 1** on the development kit programmed with the network coordinator sample to reopen the Zigbee network.
+   After some time, the devices join the Zigbee network.
+   On the development kit programmed with the light bulb sample, **LED 3** and **LED 4** turn on when the light bulb joins the network.
```
Comment (sebastiandraus) [inline doc/nrf/ug_zigbee_qsg.rst:180]: LED 4 turns on immediately after application is started, only LED3 is turned on when devices joined the network.

**Review by sebastiandraus — CHANGES_REQUESTED** [review]

File: doc/nrf/ug_zigbee_qsg.rst (hunk 1 above)
Comment (greg-fer) [reply doc/nrf/ug_zigbee_qsg.rst:10]: This is left out on purpose in the first paragraph. I don't want the reader to be distracted with a link to external page.
The "Set up the software" covers installation of VS Code extension.

**Comment by greg-fer** [comment]
@sebastiandraus , applied your edits.
@Avalei , added new figures -- please recheck.

File: doc/nrf/ug_zigbee_qsg.rst [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+   #. Click :guilabel:`Select` and then click :guilabel:`Create Application`.
+
+      .. figure:: images/zigbee_qsg_vsc_create_application_section.png
+         :alt: nRF Connect for Visual Studio Code - Create Application panel
+
+   #. If needed, click :guilabel:`Yes` when prompted if you trust the authors of the files.
+
+#. Generate the network coordinator application:
+
+   a. In the Applications panel, hover with your mouse cursor over the application name and click :guilabel:`Add Build Configuration` button that appears next to it.
```
Comment (Avalei) [inline doc/nrf/ug_zigbee_qsg.rst:163]:
```suggestion
   a. In the Applications panel, hover with your mouse cursor over the application name and click the :guilabel:`Add Build Configuration` button that appears next to it.
```

**Review by Avalei — CHANGES_REQUESTED** [review]
doc/nrf/images/zigbee_qsg_topology.svg
- ED circle has a white border, please remove it.

doc/nrf/images/zigbee_topology_types.svg
- ED circles have white borders, please remove them.

File: doc/nrf/ug_zigbee_qsg.rst [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+Overview
+********
+
+As part of this guide, we are going to program three Zigbee samples onto three different development kits in order to set up a basic Zigbee network composed of three devices, or nodes.
+A node is "a testable implementation of a Zigbee application on a single stack with a single network address, and on a single network".
+
+.. figure:: images/zigbee_qsg_topology.svg
+   :alt: Zigbee star topology setup for this guide
+
+   Zigbee star topology setup for this guide
```
[the remark this answers is not in the document]
Comment (greg-fer) [reply doc/nrf/ug_zigbee_qsg.rst:22]: This is not related to the image, but to the ToC field to the right. The ToC field overlaps with the figure area and pushes it to the left.

**Comment by greg-fer** [comment]
@Avalei , @sebastiandraus , please re-review.

**Review by Avalei — APPROVED** [review]
Looks good.

**Review by sebastiandraus — APPROVED** [review]
LGTM

**Pull request merged**